presto-docs/src/main/sphinx/functions/khyperloglog.rst

```diff
@@ -0,0 +1,75 @@
+======================
+KHyperLogLog Functions
+======================
+
+Presto implements the `KHyperLogLog <https://research.google/pubs/pub47664/>`_
+algorithm and data structure. KHyperLogLog data structure can be created
+through :func:`khyperloglog_agg` function.
+
+
+Data Structures
+---------------
+
+KHyperLogLog is a data sketch that compactly represents the association of two
+columns. It is implemented in Presto as a two-level data structure composed of
+a MinHash structure whose entries map to HyperLogLog sketches.
+
+Serialization
+-------------
+
+KHyperLogLog sketches can be cast to and from ``varbinary``. This allows them to
```

Comment:
Similar to `HyperLogLog` and `P4HyperLogLog`, create an entry with label in `language/types.rst`

```
:ref:`hyperloglog_type` can be cast to and from ``varbinary`` ...
```

Reply:
This line is not updated yet. You can replace `KHyperLogLog sketches`  with
```
:ref:`hyperloglog_type`
```